docs/semgrep-app/pricing-and-billing.mdx

```diff
@@ -1,53 +1,167 @@
+
+* Semgrep CLI
+* Semgrep CI
+* Semgrep App (through the Community tier)
+
+Semgrep CLI is free open-source software (FOSS), licensed under LGPL 2.1. It is a fast static analysis command line tool for finding bugs and enforcing code standards. Semgrep CI is a source-available Docker image for running Semgrep in CI environments. All features for both Semgrep CLI and Semgrep CI are free.
+
+Semgrep App, which builds on Semgrep CLI and CI, enables developer teams to create their own institutional code standards, enforce them, and analyze findings. Semgrep App has both free and paid tiers.
+
+## Choosing a Semgrep App tier
+
+Semgrep App has three tiers:
+
+| Tier | Price | Description |
+| ---- | ------- | -------- | 
+| **Community** | Free | For general-purpose security scanning.  |
+| **Team** | US$40 monthly per seat | For the enforcement of company-specific coding standards, private rules, and the analysis of findings.  |
+| **Enterprise** | Custom pricing | For custom solutions, including deployment into virtual private clouds (VPCs) with dedicated technical support.  |
+
+Semgrep App integrates with a GitHub or GitLab account at either the organizational or individual level. There is no limit to the number of members in an organization for all tiers.
+
+Billing depends on the number of **seats**. Any member whose code has been scanned by Semgrep App takes up a seat.
+
+Members who do **not** run any scans on their code do not need a seat and are not billed. Semgrep Team tier features apply to the entire organization, even if not all members are billed for a seat.
+
+### Comparing free and paid tier features
+
+#### Organizational management
+
+| Feature  | Community | Team | Enterprise |
+| -------  | --------- | ------- | -------- |
+| **Member limit** | ♾️ unlimited | ♾️  unlimited | ♾️  unlimited |
+| **Project limit** |♾️  unlimited | ♾️  unlimited | ♾️  unlimited |
+
+#### Findings, language support, and rules
+
+| Feature  | Community | Team | Enterprise |
+| -------  | --------- | ------- | -------- | 
+| **Findings retention** | 1 month | 1 year | 1 year+ (custom) |
+| **Community rule registry** | ✔️ yes | ✔️  yes  | ✔️  yes | 
+| **Named rules** | ✔️ yes | ✔️ yes | ✔️  yes |
+| **Private rules** | ❌ no | ✔️ yes | ✔️  yes |
+| **Custom rule messages** | ❌ no | ✔️  yes | ✔️  yes |
+| **Custom language support** | ❌ no | ❌ no | ✔️  yes |
+
+Notes:
+
+* **Named rules** are rules that have been saved through the [Playground](https://semgrep.dev/editor). The Playground is a live editor where you can write your rule patterns and test them on sample code. Named rules created by Community tier members are public and available in the Playground.
+* **Custom rule messages** can be applied to public and private rules.
+
+#### Deployment, integrations, and notifications
+
+| Feature  | Community | Team | Enterprise |
+| -------  | --------- | ------- | -------- | 
+| **Automatic CI/CD** | ✔️ yes | ✔️  yes  | ✔️  yes | 
```

Comment:
I’m not quite sure what is meant by “Automatic CI/CD” — is it automatic scanning?

Reply:
This was intended to be a shortened form of "Automatic CI/CD integration" from the pricing page. I realize it lost its context. Have corrected this by completing it as above and by adding the following description: "**Automatic CI/CD integration means that repositories added to Semgrep are scanned as part of the code repository's CI pipeline when a pull request is made.**"